### ISSUE TITLE
Tooltip for Filament Settings/Cooling/Disable fan

### ISSUE THREAD

**Issue by ruggb** [alexrj/Slic3r#3846]
### Version
1.3.0 dev

### Operating system type + version
Windows 10 1607

### Behavior
Tooltip for Filament Settings/Cooling/Disable fan for the first...________...layers
reads "You can set this to a positive value to disable fan at all during the first layers, so that is does not make adhesion worse. (default 3)"

Might be better worded.........
"This disables the fan completely for the first # of layers to aid in the adhesion of media to the bed. (default 3)"

**Comment by lordofhyphens** [comment]
The text lives in PrintConfig.cpp, make a pull request ;)

Please check out the i18n branch and also edit the corresponding msgid in all the .po files.

**Comment by ruggb** [comment]
You are making a big assumption that I know what I am doing!
But I will be happy to look and see if I can figure it out.

**Comment by lordofhyphens** [comment]
Yep, this is pretty easy (hint: checkout the i18n branch, and do a text
search for the string you want to replace).

"Nothing unreal exists." - Kiri-kin-tha's First Law of Metaphysics.

**Comment by ruggb** [comment]
OK,these changes are committed in my fork ruggb/Slic3r. There is one for each file. patch 1, 2, & 3.

I guess that is the way it works. Now what?????????

in printconfig.cpp @ 169
//    def->tooltip = __TRANS("You can set this to a positive value to disable fan at all during the first layers, so that it does not make adhesion worse.");
169       def->tooltip = __TRANS("This disables the fan completely for the first # of layers to aid in the adhesion of media to the bed. (default 3)");

..............................
in en_US.po @ 289
      //**"You can set this to a positive value to disable fan at all during the first "
      //**"layers, so that it does not make adhesion worse."
289   "This disables the fan completely for the first # of layers "
290   "to aid in the adhesion of media to the bed. (default 3)");

and @ 291
//**"You can set this to a positive value to disable fan at all during the first "
//**"layers, so that it does not make adhesion worse."
292   "This disables the fan completely for the first # of layers "
293   "to aid in the adhesion of media to the bed. (default 3)");

**Comment by xoan** [comment]
https://help.github.com/articles/fork-a-repo/#next-steps

**Comment by lordofhyphens** [comment]
1) Create a branch
2) make sure branch has your work in it
3) make a github pull request (in this case, the first entry in the list is
i18n branch, the second is your branch).

**Comment by lordofhyphens** [comment]
4) Add "Fixes #3846" to the pull request description

**Comment by ruggb** [comment]
OK I think I have done all the damage I can do. I don't know where #38**5**6 came from above.

**Comment by alexrj** [comment]
Until i18n is not merged, people would need to do two PRs for each change... maybe we can ignore i18n for now, and update it just before merging?
